Makefile

```diff
@@ -0,0 +1,66 @@
+all: build test check-coverage lint  ## test, check coverage and lint
```

Comment:
need to update the help comment to add "build"

Reply:
Done, thank you!